ui/app/AppLayouts/Profile/controls/ProfileShowcasePanelDelegate.qml

```diff
@@ -0,0 +1,30 @@
+import QtQuick 2.15
+
+import utils 1.0
+
+// Component implementing the common ShowcaseDelegate interface needed for the ProfileShowcasePanel
+// Workaround until the all plafroms support public inline components.
+// Linux and windows needs to be upgraded to qt 5.15.8+
+// https://bugreports.qt.io/browse/QTBUG-89180
+
+// TODO: Revert this commit once the inline componets are supported
+// Remove this file and use the inline component once all platforms are upgraded to 5.15.8+
```

Comment:
I think it's fine even in the longterm to keep this a separate component; remember ppl might still run it under an older Qt version when compiled from source